test/ReverseProxy.Tests/Service/HealthChecks/TransportFailureRateHealthPolicyTests.cs

```diff
@@ -206,7 +207,7 @@ private HttpContext GetFailedRequestContext(ProxyError error)
         private ClusterInfo GetClusterInfo(string id, int destinationCount, double? failureRateLimit = null, TimeSpan? reactivationPeriod = null)
         {
             var metadata = failureRateLimit != null
-                ? new Dictionary<string, string> { { TransportFailureRateHealthPolicyOptions.FailureRateLimitMetadataName, failureRateLimit.ToString() } }
+                ? new Dictionary<string, string> { { TransportFailureRateHealthPolicyOptions.FailureRateLimitMetadataName, failureRateLimit?.ToString(CultureInfo.InvariantCulture) } }
```

Comment:
```suggestion
                ? new Dictionary<string, string> { { TransportFailureRateHealthPolicyOptions.FailureRateLimitMetadataName, failureRateLimit.ToString(CultureInfo.InvariantCulture) } }
```
It's in the `failureRateLimit != null` branch. Or am I missing something?

Reply:
It would have to be `.Value.ToString` in that case since it's `Nullable<double>` without that ToString overload. `?` seems cleaner (shorter).